src/main/java/stevie/StevieParser.java

```diff
@@ -0,0 +1,100 @@
+package stevie;
+
+import stevie.command.*;
```

Comment:
Maybe can change this to explicit imports? Like,
 ```java
// current
import stevie.command.*;

// changed to explicit import
import stevie.command.Command;
```

Changing this would satisfy the rule "Imported classes should always be listed explicitly." in the [Java coding standard](https://se-education.org/guides/conventions/java/intermediate.html#naming)

Reply:
Thanks for pointing that out to me. I will take note of the import statements from now on.